lua/edgy/config.lua

```diff
@@ -38,6 +38,8 @@ local defaults = {
   },
   -- enable this to exit Neovim when only edgy windows are left
   exit_when_last = false,
+  -- close edgy when all windows are hidden instead of opening one of them
+  close_when_all_hidden = false,
```

Comment:
Can you make this the default?

It actually does make a lot of sense to behave this way

Reply:
Totally, sounds good!